lib/logstash/outputs/elasticsearch/http_client_builder.rb

```diff
@@ -26,7 +24,25 @@ def self.build(logger, hosts, params)
       common_options[:timeout] = params["timeout"] if params["timeout"]
 
       if params["path"]
-        client_settings[:path] = "/#{params["path"]}/".gsub(/\/+/, "/") # Normalize slashes
+        client_settings[:path] = "/#{params["path"]}/".gsub(/\/+/, "/")
+      end
+
+      common_options[:bulk_path] = if params["bulk_path"]
+         "/#{params["bulk_path"]}".gsub(/\/+/, "/")
```

Comment:
Can we split out this gsub into its own function since we're using it everywhere?